packages/Webkul/Velocity/src/Resources/assets/js/UI/components/product-card.vue

```diff
@@ -79,7 +79,7 @@
                 </a>
             </div>
 
-            <div class="product-rating col-12 no-padding" v-else>
+            <div class="product-rating col-12 no-padding" v-else-if="guestReviewStatus">
```

Comment:
All boolean logic should have a `can` or `is` prefix like `canGuestReview`.

Same as `isEnabled`, `isUsed`, `isChecked`, etc.